docs/logs/quick-start.md

```diff
@@ -0,0 +1,102 @@
+title: Sematext Logs Quick Start
+description: Cloud based SaaS / On-premises logging as a service platform. Index machine data, search, correlate and investigate. Build in reports and dashboards with monitoring and alerting capabilities.
+
+After you get logged into Sematext Cloud at <https://apps.sematext.com> (or <https://apps.eu.sematext.com> if using Sematext Cloud Europe), the first step is to create a Logs App. An App is an independent namespace for your data.
+
+For example, if you have a development and a production environment, it might make sense to have one App for each. You can create as many Apps as you want.
+
+## Creating a Logs App
+
+You create an App by pressing the **+ Create Logs App** button in the Logs tab.
+
+![Create a new Logs App](../images/guide/logs/sematext-logs-app-create.png)
+
+Add a name and press **Continue**.
+
+![Create a new Logs App Modal](../images/guide/logs/sematext-logs-app-create-modal.png)
+
+After creating a Logs App you will see an **integrations** screen that tells you how to [send data to your new Logs App](../logs/sending-log-events). 
+
+![Logs App Elasticsearch integration](../images/guide/logs/sematext-app-logs-elasticsearch.png)
+
+Once you start sending data, you can start [searching and analyzing these logs](../logs/searching-log-events) in the Logs App or [explore your data with Kibana](../logs/kibana).
+
+![Searching Log Events](../images/logs/logsene-ui.png)
+
+### Adding Data to Your Logs App
+
+There are two ways to send data to your Logs App: 
```

Comment:
Maybe: There are two ways ingestion endpoints to which you can send data to your Logs App

Reply:
Edited.